src/nfts.ts

```diff
@@ -0,0 +1,6 @@
+import { defaultAbiCoder } from "@ethersproject/abi";
+import { BytesLike } from "@ethersproject/bytes";
+
+export function makeTokenIdExitMetadata(tokenId: number): BytesLike {
```

Comment:
Perhaps a comment somewhere explaining the link between NFTs and ERC721 and ERC1155?

Am I right in thinking that this method would still be useful for describing a _fungible_ ERC1155 token? In that case, is `nfts.ts` a good filename?

Reply:
Renamed to `token-id-metadata.ts`. Very explicit and no reference to NFTs. Will add description in README of how 721 and 1155 relate to NFTs